src/app/components/team/TeamDetails.js

```diff
@@ -223,8 +223,22 @@ const TeamDetails = ({
             { shortenOutgoingUrls ?
               <Alert
                 type="warning"
-                title={<FormattedMessage id="teamDetails.warnTitle" defaultMessage="All links sent via Check are managed" description="Text displayed in a warning box on team details page when link shortening is on" />}
-                content={<FormattedMessage id="teamDetails.warnContent" defaultMessage="Links processed in the Check platform will be rewritten to match this example: https://chck.media/x1y2z3w4. The unique code in each link will redirect your users to the original final destination." description="Text displayed in a warning box on team details page when link shortening is on" />}
+                title={<FormattedMessage id="teamDetails.warnTitle" defaultMessage="All links sent via Check will be rewritten." description="Text displayed in the title of a warning box on team details page when link shortening is on" />}
+                content={
+                  <FormattedMessage
+                    id="teamDetails.warnContent"
+                    defaultMessage="The rewritten links will also include the UTM code if provided. The UTM code does not change the destination URL.{br}{br}{before} https://www.example.com/your-link{br}{after} https://chck.media/x1y2z3w4/{code}"
+                    values={
+                      {
+                        after: <strong>AFTER:</strong>,
+                        before: <strong>BEFORE:</strong>,
```

Comment:
AFTER and BEFORE won't be translated. Perhaps we can use FormattedHTMLMessage instead?

Reply:
thanks @amoedoamorim that is an obvious improvement. Take a look and see if thats what you were thinking with my last commit